docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -622,6 +622,16 @@ replace()
 Replaces a header. *key* is a string that supplies the header key. *value* is a string that supplies
 the header value. If the header does not exist, it is added as per the *add()* function.
 
+setHttp1ReasonPhrase()
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: lua
+
+  headers:setHttp1ReasonPhrase(reasonPhrase)
+
+Sets a custom HTTP1 response reason phrase. This call is *only valid in the response flow*.
+*reasonPhrase* is a string that supplies the reason phrase value.
```

Comment:
```suggestion
Sets a custom HTTP/1 response reason phrase. This call is *only valid in the response flow*.
*reasonPhrase* is a string that supplies the reason phrase value. Additionally this call only
effects HTTP/1 connections. It will have no effect if the client is HTTP/2 or HTTP/3.
```